app/views/api/integrations/apicast/shared/_authentication_settings.html.slim

```diff
@@ -1,5 +1,6 @@
 = f.toggled_inputs 'Authentication Settings' do
   - if @service.oidc?
+    = f.input :oidc_issuer_type, hint: true, as: :select, collection: Proxy.oidc_issuer_types
```

Comment:
Default will be keycloak which will come pre-selected right?

Reply:
It can be.

Reply:
I guess it depends a bit on what we expect most people to use. Given currently we only allow keycloak and keycloak being the redhat way of doing it seems defaulting to keycloak makes sense.